CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # [Versions](https://mui.com/versions/)
 
+## 6.0.0-rc.0
+
+<!-- generated comparing v6.0.0-beta.6..next -->
+
+_Aug 21, 2024_
+
+A big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚡ Rendering performance improvements
```

Comment:
@romgrk is this accurate?

Reply:
Yes, lgtm

Reply:
I did a quick run of https://github.com/mui/material-ui/tree/next/benchmark/browser#output

Before
```
Before

noop (baseline):
  02.51 ±00.19ms
Grid (html):
  31.24 ±00.83ms
Grid System:
  258 ±8%
Grid Material UI:
  247 ±12%
Table:
  138 ±5%
React primitives:
  53 ±2%
React components:
  58 ±2%
Styled MUI:
  101 ±4%
Styled emotion:
  81 ±1%
Styled SC:
  76 ±4%
makeStyles:
  84 ±2%
Box Baseline:
  93 ±2%
Box MUI:
  152 ±5%
Box Theme-UI:
  141 ±15%
Box Chakra-UI:
  124 ±12%
styled-components Box + @mui/system:
  148 ±3%
styled-components Box + styled-system:
  114 ±2%
```

After
```
After

noop (baseline):
  02.65 ±00.61ms
Grid (html):
  29.95 ±03.57ms
Grid System:
  181 ±6%
Grid Material UI:
  173 ±6%
Table:
  109 ±7%
React primitives:
  57 ±2%
React components:
  60 ±2%
Styled MUI:
  95 ±5%
Styled emotion:
  88 ±3%
Styled SC:
  82 ±3%
makeStyles:
  88 ±5%
Box Baseline:
  95 ±2%
Box MUI:
  157 ±2%
Box Theme-UI:
  141 ±3%
Box Chakra-UI:
  123 ±2%
styled-components Box + @mui/system:
  155 ±4%
styled-components Box + styled-system:
  118 ±3%
```

the Grid and Table seems much faster, great to see 👍

Reply:
Interesting, I wasn't able to optimize the grid much due to some issues with prop access.

By the way, would anyone know why [this section](https://github.com/mui/material-ui/blob/90c4624f57efa7a557589bee4cd8a799484e161f/packages/mui-material/src/Grid/Grid.js#L312-L329) uses prop access for styling instead of variants?

Reply:
I guess we don't care about this Grid, since Grid2 is the new stable one: https://next.mui.com/material-ui/migration/migration-grid-v2/#with-material-ui-v6.

Reply:
Also, I will be curious to rerun the performance benchmark of Pigment CSS vs. Emotion in https://deploy-preview-43438--material-ui.netlify.app/blog/introducing-pigment-css/#better-runtime-performance.